openmdao/docs/basic_guide/basic_case_recording.rst

```diff
@@ -0,0 +1,26 @@
+.. _basic_case_recording:
+
+************************
+Basic Recording Example
+************************
+
+Recording Terminology
+---------------------
+
+| **Case**: A Case stores a snapshopt of all the variable values, metadata, and options of a model, or a sub-set of a model, at a particular point in time
+| **Case Recorder**: An OpenMDAO module used to store the state of a `Problem` before, during, or after execution in an SQL file.
+| **Sources**: The OpenMDAO objects responsible for recording the case. `Problem`, `Driver`, and `System`.
```

Comment:
the possible sources are:  `Problem`, `Driver`,  a system (by pathname) or a solver (by pathname)
per the docstring:  The unique id of the system/solver/driver/problem that did the recording.

this may be a good place to use `.. autoclass::` to describe these classes